fedmsg.d/fmn.py

```diff
@@ -19,11 +19,19 @@
     "fmn.rules.utils.use_pkgdb2": False,
     "fmn.rules.utils.pkgdb2_api_url": "http://209.132.184.188/api/",
     "fmn.rules.cache": {
```

Comment:
My first thoughts where: we probably want to keep the old default, but now iirc it doesn't work if you're not using redis.